man/sysupdate.d.xml

```diff
@@ -595,17 +595,26 @@
 
         <para>If the source type is selected as <constant>url-file</constant> or
         <constant>url-tar</constant> this must be a HTTP/HTTPS URL. The URL is suffixed with
-        <filename>/SHA256SUMS</filename> to acquire the manifest file, with
-        <filename>/SHA256SUMS.gpg</filename> to acquire the detached signature file for it, and with the file
-        names listed in the manifest file in case an update is executed and a resource shall be
-        downloaded.</para>
+        the value assigned to the <varname>Manifest</varname> variable to acquire the manifest file. If the
+        manifest name is <filename>/SHA256SUMS</filename> the detached signature file for it will be acquired (if
+        verification is enabled),and with the file names listed in the manifest file in case an update is
+        executed and a resource shall be downloaded.</para>
 
         <para>For all other source resource types this must be a local path in the file system, referring to
         a local directory to find the versions of this resource in.</para>
 
         <xi:include href="version-info.xml" xpointer="v251"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>Manifest=</varname></term>
+
+        <listitem><para>Specifies the filename of the manifest. Defaults to <filename>/SHA256SUMS</filename>.
+        Overriding the <varname>Manifest</varname> disables verification.</para>
+
+        <xi:include href="version-info.xml" xpointer="v257"/></listitem>
+      </varlistentry>
```

Comment:
i think we should have some language here that strongly suggests that people should please stick to SHA256SUMS for this, and stop playing games with everyone making up their own random names...